project/cmake/CMakeLists.txt

```diff
@@ -24,7 +24,6 @@ option(ENABLE_PLIST       "Enable AirPlay support?" ON)
 option(ENABLE_NFS         "Enable NFS support?" ON)
 option(ENABLE_AIRTUNES    "Enable AirTunes support?" ON)
 option(ENABLE_CEC         "Enable CEC support?" ON)
-option(ENABLE_OPENSSL     "Enable OpenSSL?" ON)
```

Comment:
@wsnipex: openssl is not used on Windows, so we keep the option in line 41, right?